public/js/src/module/utils.js

```diff
@@ -108,13 +108,40 @@ function getTitleFromFormStr(formStr) {
     return matches && matches.length > 1 ? matches[1] : null;
 }
 
+/**
+ * @param {string} csv
+ */
 function csvToArray(csv) {
+    const input = csv.trim();
     const options = {
         skipEmptyLines: true,
     };
-    const result = Papa.parse(csv.trim(), options);
+
+    let result = Papa.parse(input, options);
+    const { errors } = result;
+
+    if (errors[0]?.code === 'UndetectableDelimiter' && errors[1] == null) {
```

Comment:
`errors[1] == null` feels like it relies on JS quirks whereas a check that `errors` has a length of 1 would better match the intent.